modular_skyrat/modules/antag_optin/code/mind.dm

```diff
@@ -0,0 +1,22 @@
+/datum/mind
+	/// The optin level set by preferences.
+	var/ideal_opt_in_level
+
+/datum/mind/transfer_to(mob/new_character, force_key_move)
```

Comment:
note i only put this on transfer_to() becauase i couldnt find a better proc for it, ideally this would be set on something as soon as a character is generated but mind new() or mob new() dont have a client yet